clang/test/CodeGenHLSL/builtins/dst.hlsl

```diff
@@ -0,0 +1,57 @@
+// RUN: %clang_cc1 -finclude-default-header -x hlsl -triple dxil-pc-shadermodel6.2-library %s -fnative-half-type -emit-llvm -disable-llvm-passes -o - | FileCheck %s
+
+
+// CHECK-LABEL: linkonce_odr noundef nofpclass(nan inf) <4 x float> @_ZN4hlsl8__detail8dst_implIfEEDv4_T_S3_S3_(

```

Comment:
These are far too specific and will probably break in the future. Please simplify to the minimum thing that you need to check. In this case that is the following:
- the `define` keyword
- the expected type (<4 x float>)
- the demangled function name (dst_impl) 

Something like
```
// CHECK-LABEL: define {{alpha/space/paren regex}} <4 x float> @{{alpha/num/under regex}}dst_impl{{regex}}(
```

All the `CHECK-LABEL`s should be updated